docs/2_4_common_schema.adoc

```diff
@@ -813,7 +813,9 @@ A tool may ignore this information. +
 _[However, it is convenient for a user that <<startTime>>, <<stopTime>>, <<tolerance>> and <<stepSize>> have already a meaningful default value for the model at hand.]_ +
 <<startTime>>, <<stopTime>> and <<stepSize>> refer to the unit of the <<independent>> variable.
 <<stepSize>> defines the preferred <<communicationStepSize>> for Co-Simulation.
-For Model Exchange and Scheduled Execution <<stepSize>> has no defined meaning.
+For Model Exchange and Scheduled Execution <<stepSize>> has no defined meaning. 
+_[This does not prohibit the use of <<stepSize>> in the context of Model Exchange. 
+For example, a fixed step size solver can use it as a default step size, but since the required <<stepSize>> depends on the solver itself a defined meaning of <<stepSize>> for Model Exchange is impossible.]_
```

Comment:
```suggestion
For Model Exchange and Scheduled Execution <<stepSize>> has no defined meaning. 
_[This does not prohibit the use of <<stepSize>> in the context of Model Exchange._ 
_For example, a fixed step size solver can use it as a default step size, but since the required <<stepSize>> depends on the solver itself a defined meaning of <<stepSize>> for Model Exchange is impossible._
Or it could be used as a result grid to be used for the outputs in case of a variable step solver]_
```